docs/src/routes/generated/Routes.js

```diff
@@ -166,4 +178,14 @@ export const guidelines = [{
         content={STRATEGY}
       />
     )
+  },{
+    path: ROUTES.GENERATED_GUIDELINES_TONE_OF_VOICE,
+    component: () => (
+      <MarkdownPage
+        fileName="tone-of-voice.md"
+        title="Tone of voice"
+        subtitle="Our tone of voice is the unuqie way in which we speak to travellers, partners and ourselves."
```

Comment:
```suggestion
        subtitle="Our tone of voice is the unique way in which we speak to travellers, partners and ourselves."
```

This is why you copy paste and don't just type it manually 🤦